src/style.css

```diff
@@ -1,3 +1,76 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
Don't use the `*` selector as it impacts the performance. Set styles only for elements that require them. Zeroing out your margins, paddings, or other styles with `*` is still inefficient for the browser.